README.md

```diff
@@ -0,0 +1,34 @@
+## HOWTO build
+
+* fetch sources
+```
+git clone -b morty git://git.yoctoproject.org/poky.git
+git clone -b morty git://git.openembedded.org/meta-openembedded.git
+git clone -b morty git@github.com:prplfoundation/meta-prplmesh.git
```

Comment:
Your build instructions should use the https URL so you don't require a github account to be able to clone it.

Reply:
Done